cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -103,6 +103,24 @@ <h2 id="previewer">Preview</h2>
         {%include "cernopendata_ispy/ispy_standalone.html"%}
     {% endif %}
 
+    {% if record.cross_section %}
+    <h2>Cross section</h2>
+        {% if record.cross_section.description %}
+            <span>{{record.cross_section.description | safe}}</span>
+            <br>
+        {% else %}
+            <p>For {{record.collision_information.type | safe}} collisions at {{record.collision_information.energy | safe}}, this sample has a cross section of (<span><strong>{{record.cross_section.total_value | safe}}</strong></span> +/- <span><strong>{{record.cross_section.total_value_uncertainty | safe}}</strong></span>) pb, calculated using the method described in <a href="/docs/cms-guide-cross-sections">CMS guide for cross sections</a>.</p>
+        {% endif %}
+        {% if record.cross_section.matching_efficiency %}
+            <p>This cross section takes into account a matching efficiency of <span><strong>{{record.cross_section.matching_efficiency | safe}}</strong></span> and a filtering efficiency of <span><strong>{{record.cross_section.filter_efficiency | safe}}</strong></span>, based on generator settings and/or filters.</p>
+        {% else %}
+            <p>This cross section takes into account a filtering efficiency of <span><strong>{{record.cross_section.filter_efficiency | safe}}</strong></span>, based on generator settings and/or filters.</p>
+        {% endif %}
+        {% if record.cross_section.neg_weight_fraction %}
+            <p>If this sample was generated at NLO, it has <span><strong>{{record.cross_section.neg_weight_fraction | safe}}</strong></span>% events with negative weights.</p>
+        {% endif %}
```

Comment:
Add
<p>This is a leading-order value, and cross sections evaluated at next-to-leading order (NLO) or next-to-next-to-leading order (NNLO) are preferred, if available.</p>

Reply:
Added.